modules/Sources/Generated/XCAssets+Generated.swift

```diff
@@ -305,6 +306,34 @@ public extension ColorAsset.SystemColor {
   }
 }
 
+public struct DataAsset {
```

Comment:
It's unclear to me what this is for @LukasKorba; it seems auto-generated but I don't see any changes that would have triggered it.

Reply:
This is ok, swiftgen has a stencil file that is checking several possible asset types and since project has now a new type of generic data, it must create DataAsset for it, same like ImageAsset, FontAsset, etc.